app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm

```diff
@@ -184,6 +200,13 @@ update message ({ delegatebw, totalQuantity, distributionRatio, stakeAmountModal
         OpenStakeAmountModal ->
             ( { model | isStakeAmountModalOpened = True }, Cmd.none )
 
+        SubmitAction ->
+            let
+                cmd =
+                    { delegatebw | from = accountName, receiver = accountName } |> Action.Delegatebw |> encodeAction |> Port.pushAction
```

Comment:
You may want to clear model after submitting action.

Reply:
OK I will do it at next PR :)